packages/kbn-dev-utils/src/ci_stats_reporter/ci_stats_reporter.ts

```diff
@@ -80,6 +81,15 @@ export class CiStatsReporter {
     const timings = options.timings;
     const upstreamBranch = options.upstreamBranch ?? this.getUpstreamBranch();
     const kibanaUuid = options.kibanaUuid === undefined ? this.getKibanaUuid() : options.kibanaUuid;
+    let email;
+
+    try {
+      const { stdout } = await execa('git', ['config', 'user.email']);
```

Comment:
Do we want to specify cwd here?

Reply:
I didn't find it necessary, even when calling it from outside the Kibana root. We would have to make some changes to `@kbn/pm` to be able to use something that depends on `REPO_ROOT` from `@dev/utils` because it attempts to load `package.json`. I tried adding `json-loader` but it didn't seem that strait-forward.

Reply:
Okay, it's a pretty solid edge case. We detect the Kibana root differently in kbn/pm and could just pass that in if we wanted... but it's not important